src/renderer/helpers/rx/fromPromise.test.ts

```diff
@@ -37,10 +37,10 @@ describe('/helpers/rx/fromPromise', () => {
     })
 
     it('should trigger onError cb ', (done) => {
-      const targetStream$ = fromPromise$(
+      const targetStream$ = fromPromise$<number, number>(
         () => Promise.reject(3),
         1,
-        (e) => e * 2
+        (_) => 6
```

Comment:
here we check that argument was passed. imo its better to use argument in test too

Reply:
`(e) => e * 2` can't work, since we have `e: Error`

Reply:
u r right. it's a lack of implementation typings. lets leave thing this way for now